README.md

```diff
@@ -52,6 +52,7 @@ Name|Description
 [custom-addon/ntp-sync](kubermatic-addons/custom-addon/ntp-sync) | DaemonSet to execute `ntpdate primary secondary` scheduled on every node of a cluster
 [custom-addon/docker-pull](kubermatic-addons/custom-addon/docker-pull) | DaemonSet to pull e.g. `docker.io` based images on every node with a docker-secret, to prevent rate-limited infrastructure pods.
 [custom-addon/kube-proxy-ipvs-patch](kubermatic-addons/custom-addon/kube-proxy-ipvs-patch) | Custom overwrite Addon to patch IPVS mode to `strictARP: true`.
+[custom-addon/flux2](kubermatic-addons/custom-addon/argocd-ha) | Deploys [ArgoCD](https://github.com/argoproj/argo-cd/) - a declarative, GitOps continuous delivery tool for Kubernetes (as high availability installation).
```

Comment:
flux2 -> argocd

Reply:
and I would avoid `-ha` detail ... let's consider that as a default option that stuff is being as HA in k8s ;)